package.json

```diff
@@ -8,6 +8,8 @@
     "url": "git://github.com/twingly/twingly-search-api-node.git"
   },
   "dependencies": {
+    "chai-datetime": "^1.4.0",
```

Comment:
Indeed they should, I used `--save` when I should have used `--save-dev`, will fix, thanks